source/tutorials/fmt-cdiff-khanna.rst

```diff
@@ -400,6 +400,87 @@ For more information about animated ordinations, visit Emperor's `documentation`
 
 .. _`fmt cdiff taxonomy`:
 
+
+Longitudinal analysis
+---------------------
+
+For longitudinal studies, we can use the :doc:`q2-longitudinal plugin <longitudinal>` to evaluate the association between experimental observations, treatments, and time.
+
+We’ll start by using the ``first-distances`` method to examine how FMT reshapes an individual's gut microbiome to resemble that of the donor.
+
+.. command-block::
+
+  qiime longitudinal first-distances \
+    --i-distance-matrix core-metrics-results/unweighted_unifrac_distance_matrix.qza \
+    --m-metadata-file sample_metadata.tsv \
+    --p-state-column timepoint \
+    --p-individual-id-column donor_group \
+    --p-baseline -99 \
+    --p-replicate-handling random \
+    --o-first-distances core-metrics-results/unweighted_unifrac_first_distances.qza
+
+Next, we will use the ``volatility`` visualizer to interactively examine temporal changes in distance from donor, alpha diversity, and PCoA ordinations for each subject. We will drop two samples that were collected long before FMT treatment.
+
+.. command-block::
+
+  egrep -v '(10057.227.3|10057.227.2)' sample_metadata.tsv | grep 'atient' > sample_metadata_select.tsv

```

Comment:
Will it be possible to add an explanation of this line and why we need it?

Reply:
👍